docs/python.md

```diff
@@ -1,3 +1,33 @@
 # Awesome Open Source - Python
 
-Coming Soon!
+- [APScheduler](https://github.com/agronholm/apscheduler) - A lightweight, in-process task scheduler.
+- [Arrow](https://github.com/arrow-py/arrow) - Better dates & times for Python.
+- [Bokeh](https://github.com/bokeh/bokeh) - Interactive web plotting for Python.
+- [Crossbar](https://github.com/crossbario/crossbar/) - Open-source Unified Application Router (Websocket & WAMP for Python on Autobahn).
+- [Datasette](https://github.com/simonw/datasette) - An open source multi-tool for exploring and publishing data.
+- [dejavu](https://github.com/worldveil/dejavu) - Audio fingerprinting and recognition.
+- [djedi-cms](http://djedi-cms.org/) - A lightweight but yet powerful Django CMS with plugins, inline editing and performance in mind.
+- [Dramatiq](https://github.com/Bogdanp/dramatiq) - A fast and reliable background task processing library for Python 3.
+- [FastAPI Users](https://github.com/fastapi-users/fastapi-users) - Ready-to-use and customizable users management for FastAPI.
+- [Faker](https://github.com/joke2k/faker) - Faker is a Python package that generates fake data for you.
+- [fuzzywuzzy](https://github.com/seatgeek/fuzzywuzzy) - Fuzzy String Matching.
```

Comment:
this project moved to: https://github.com/seatgeek/thefuzz

could you please update to the latest?